hours/tests/test_resource_periods_as_text.py

```diff
@@ -586,6 +628,97 @@ def test_date_period_as_text_with_time_span_resource_state_no_opening_hours(reso
     )
 
 
+@pytest.mark.django_db
+def test_resource_date_periods_as_text_ordering(resource):
+    """Test that the DatePeriods are ordered correctly in the cotent."""
+    order = [
+        ("2", None, None),
+        ("1", None, None),  # Created before 2
```

Comment:
Maybe create a separate test case for these tie-breakers; it's not exactly clear to me how it should behave on this situation (sorted by id or name?). Separating this to its own test case should also make this a bit more readable (no need for any mental gymnastics when reading the test data)